### PR TITLE
:sparkles: Upgrades SIMPL to Django to 2.2

### LINKED ISSUE #15: Upgrade to Django 2.2
The Django 1.11 LTS support ends April 2020. We need to upgrade to Django 2.2, which currently has mainstream support and will have LTS until April 2022.

### LINKED ISSUE #18: Support Python 3.7
I _think_ this just needs a more recent version of gevent, which has supported Python 3.7 since https://github.com/gevent/gevent/pull/1069

### PR DESCRIPTION
**Pull request by jefftriplett**
Fixes #15 (Django to 2.2)

Fixes #18 (Python 3.7 support)

This is ready for review.

### CODE REVIEW

**Comment by williln** [comment]
LGTM! 🎉

File: config/urls.py [hunk 1]
```diff
@@ -2,45 +2,43 @@
...
-    url(r'^accounts/', include('allauth.urls')),
-]
 
-# Static Media and User Uploads
-urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT, show_indexes=True)
-urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Simpl Games API",
+        default_version='v0.7.11',
```
Comment (janeeisenstein) [inline config/urls.py]: Has .bumpversion.cfg been edited to change this default_version?

File: config/urls.py [hunk 2]
```diff
@@ -2,45 +2,43 @@
...
-    url(r'^apis/', include(api_router.urls, namespace='simpl_api')),
-    url(r'^apis/',
-        include(simpl_users_api_router.urls, namespace='simpl_users_api')),
-    url(r"^apis/hooks/",
-        include("thorn.django.rest_framework.urls", namespace="webhook")),
-    url(r'^apis/bulk/',
-        include(bulk_api_router.urls, namespace='simpl_bulk_api')),
-    url(r'^$', schema_view),  # Swagger
+urlpatterns = [
+    path('users/', include(('simpl_users.urls', 'name'))),
```
Comment (janeeisenstein) [inline config/urls.py]: Shouldn't 'name' be "users"?

File: requirements.txt [hunk 3]
```diff
@@ -2,37 +2,33 @@
 # like Pillow and psycopg2
 # See http://bitly.com/wheel-building-fails-CPython-35
 # Verified bug on Python 3.5.1
-wheel==0.29.0
+wheel==0.33.4
 
 # Django
-Django>=1.11.18,<2.0
+Django<2.3
```
Comment (janeeisenstein) [inline requirements.txt]: Don't we need a >=2.2 to bracket this?

File: config/urls.py (hunk 2 above)
Comment (jefftriplett) [reply config/urls.py]: Good catch. /me pushes a fix

File: config/urls.py (hunk 1 above)
Comment (jefftriplett) [reply config/urls.py]: :tractor: Refactored and bumpversion not update simpl.__version__ (stored in `simpl/__init__.py`) which gets imported via `config/urls.py`. This cleans it up a lot. Good catchup.

**Comment by janeeisenstein** [comment]
@jefftriplett This branch's tests are failing. Please fix.

File: config/urls.py [hunk 4]
```diff
@@ -2,45 +2,43 @@
...
+        # license=openapi.License(name="GNU GENERAL PUBLIC LICENSE"),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+)
 
 # Our application urls
-urlpatterns += [
-    url(r'^users/', include('simpl_users.urls', namespace='users')),
-
```
Comment (janeeisenstein) [inline config/urls.py]: Why did you remove the simpl_users.urls? Aren't they needed for django admin?

File: requirements.txt [hunk 5]
```diff
@@ -49,41 +45,38 @@ django-redis==4.8.0
 # Celery support (needed for Thorn)
 celery<=4.2
 
-thorn==1.5.0
-
-# Wharton styles
-git+https://github.com/jefftriplett/django-base-theme.git@432f87f  # Pinned to the latest
-django-bootstrap3==8.2.1
+# thorn==1.5.2
+git+https://github.com/simplworld/thorn.git@upgrade-to-django-2-dot-2  # Our fork until the main project is upgraded to Django 2.2
```
Comment (janeeisenstein) [inline requirements.txt]: This repo's README states:
```
Thorn version 1.0 runs on
    Python (2.7, 3.4, 3.5)
    PyPy (5.1.1)
    Jython (2.7).
    Django (1.8, 1.9, 1.10)
```
Is that accurate?

File: config/urls.py (hunk 4 above)
Comment (jefftriplett) [reply config/urls.py]: This is part of the change you asked me to make and is dependent on: This is part of https://github.com/simplworld/simpl-users/pull/8

The Rest API endpoints are on #L36:

```python
path('apis/', include((simpl_users_api_router.urls, "simpl_users_api"))),
```

File: requirements.txt (hunk 5 above)
Comment (jefftriplett) [reply requirements.txt]: Our upstream pull request was accepted. 🎉  https://github.com/robinhood/thorn/pull/38

So I think I can update and remove our forked copy which will resolve this. I'll update after my meetings this morning.

Comment (jefftriplett) [reply requirements.txt]: The Thorn project accepted our PR so I deleted our fork which resolves this.
